tests/v2/validation/snapshot/snapshot.go

```diff
@@ -115,49 +120,44 @@ func snapshotRestore(t *testing.T, client *rancher.Client, clusterName string, e
 	require.Equal(t, initialIngressName, ingressResp.ObjectMeta.Name)
 
 	if isRKE1 {
-		initialSnapshotValues := snapshotRKE1(t, client, podTemplate, deployment, clusterName, clusterID, localClusterID, etcdRestore, isRKE1)
-		restoreRKE1(t, client, initialSnapshotValues, etcdRestore, clusterName, clusterID)
+		cluster, initialSnapshotValues := snapshotRKE1(t, client, podTemplate, deployment, clusterName, clusterID, localClusterID, etcdRestore, isRKE1)
+		restoreRKE1(t, client, initialSnapshotValues, etcdRestore, cluster, clusterID)
 	} else {
-		initialSnapshotValues := snapshotV2Prov(t, client, podTemplate, deployment, clusterName, clusterID, localClusterID, etcdRestore, isRKE1)
-		restoreV2Prov(t, client, initialSnapshotValues, etcdRestore, clusterName, clusterID)
+		cluster, initialSnapshotValues := snapshotV2Prov(t, client, podTemplate, deployment, clusterName, clusterID, localClusterID, etcdRestore, isRKE1)
```

Comment:
assuming this is hard coded to the localClusterID because that's where snapshots live? If that's the case, why do we have to provide this as an argument if its always on the local cluster?

if this needs to stay as a variable, can you add a comment why it is the local cluster

Reply:
That can be omitted completely. Sam did some work with `VerifySnapshots` in shepherd that we can specifically just have over there and not here. I'll update that.

Reply:
Resolved.